src/fireEvent.js

```diff
@@ -17,11 +17,25 @@ const isTouchResponder = (element?: ReactTestInstance) => {
   return !!element?.props.onStartShouldSetResponder || isTextInput(element);
 };
 
+const isPointerEventEnabled = (element?: ReactTestInstance) => {
+  if (
+    element?.props.pointerEvents === 'none' ||
+    element?.props.pointerEvents === 'box-only'
```

Comment:
`box-only` means this element doesn't disable pointers, but its children do. `box-none` means this element disables pointers, but it's children won't.

So with this code, if a `View` has `box-only`, and I try to trigger a press on it, then it would return `false` here whereas it's actually pressable.

Maybe there needs to be a second argument `isParent` or something, and then it could be something like:

```js
isParent
  ? element?.props.pointerEvents === 'box-only'
  : element?.props.pointerEvents === 'box-none'
```